PHPCI/Plugin/CopyBuild.php

```diff
@@ -75,7 +75,7 @@ public function execute()
     protected function wipeExistingDirectory()
     {
         if ($this->wipe === true && $this->directory != '/' && is_dir($this->directory)) {
-            $cmd = 'rm -Rf "%s*"';
+            $cmd = 'rm -Rf "%s"*';
```

Comment:
Since there is ``ìs_dir``` chech are we sure we can manage directory where path is not resolved?
```
php > var_dump(is_dir('/tm'));
bool(false)

php > var_dump(is_dir('/tmp'));
bool(true)
```
I suppose the ```*``` is not supported.